templates/crud/generic_update_view.html

```diff
@@ -18,8 +18,10 @@
                             class="btn-submit mt-6">
                         Save
                     </button>
-                    <a href="javascript:window.history.back()"
-                       class="btn-cancel">Back</a>
+                    <a href="{% url 'pages:home' %}"
```

Comment:
this is incorrect, should go back to generic crud view

Reply:
the 4 tiles view or the form itself?

Reply:
one needs a cookie, the other is just similar to what i did